### DISCUSSION TITLE
request: conditional key remap

### DISCUSSION THREAD

**Discussion by vincehi** [rxhanson/Multitouch-Community#101] [Ideas]
Hi, something I would find great would be to be able to remap the keys and shortcuts depending on the application you are using. Indeed, in the terminal I want to use the command key to have the same shortcuts as my system. The alternative would be to use karabiner but why not have everything in the same application?
Greet

**Comment by rxhanson** [comment]
This is on my roadmap, but I don't have a timeframe for it.
